distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalogValidation.java

```diff
@@ -76,149 +69,129 @@
   private static final String ERROR_METACARD =
       XML_RECORD_RESOURCE_PATH + "/sampleErrorMetacard.xml";
 
-  private static final String CSW_RECORD_XPATH =
-      "/GetRecordsResponse/SearchResults/Record[identifier=\"%s\"]";
-
   @BeforeExam
   public void beforeExam() {
     try {
       waitForSystemReady();
       getServiceManager().startFeature(true, SAMPLE_VALIDATOR);
+
+      // start test with validation errors/warnings allowed in catalog/search results
+      configureValidationMarkerPlugin(
```

Comment:
❓ How are these configuration reset at the end?

Reply:
@clockard can probably confirm, but I was under the impression that the system configuration gets reset between test classes

Reply:
I'll put it into an `@AfterExam`, anyway

Reply:
Reset should no be just between classes. This makes for tests that are dependent on others creating bigger issues. I think we have a lot of those right now.